wodles/aws/tests/test_aws.py

```diff
@@ -14,6 +15,8 @@
 sys.modules['boto3'] = MagicMock()
 sys.modules['botocore'] = MagicMock()
 
+LOGS_PATH = os.path.join(sys.path[0], 'tests/data/log_files')
```

Comment:
Why are you setting the full path here instead of reusing the `test_data_path` variable from line 20 and appending to it `log_files`?